walkthroughs/howto-cross-ekscluster-appmesh-with-cloudmap/README.md

```diff
@@ -0,0 +1,232 @@
+                                 Dload  Upload   Total   Spent    Left  Speed
+100     4  100     4    0     0    205      0 --:--:-- --:--:-- --:--:--   210
+2: blue
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+100     3  100     3    0     0    136      0 --:--:-- --:--:-- --:--:--   142
+......
+......
+196: blue
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+100     4  100     4    0     0    236      0 --:--:-- --:--:-- --:--:--   250
+197: blue
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+100     3  100     3    0     0    180      0 --:--:-- --:--:-- --:--:--   187
+198: red
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+100     4  100     4    0     0    212      0 --:--:-- --:--:-- --:--:--   222
+199: blue
+```
+
+You may now look at our X-Ray console to see the service map and traces.
+
+## FAQ
+### 1. My front app is unable to talk to colorapp on the seond cluster?
+You need to open the port 8080 on second cluster's nodegroup SG, so the front app can talk to it.
+
+### 2. How can I use Cloud Map namespaces other than PrivateDnsNamespace?
+AWS Cloud Map supports three types of namespaces;
+1. [PublicDnsNamespace](https://docs.aws.amazon.com/cloud-map/latest/api/API_CreatePublicDnsNamespace.html): Namespace that is visible to the internet.
+2. [PrivateDnsNamespace](https://docs.aws.amazon.com/cloud-map/latest/api/API_CreatePrivateDnsNamespace.html): Namespace that is visible only in the specified VPC.
+3. [HttpNamespace](https://docs.aws.amazon.com/cloud-map/latest/api/API_CreateHttpNamespace.html): Namespace that supports only HTTP discovery using DiscoverInstances. This namespace does not support DNS resolution.
+
+Currently App Mesh only supports backend applications running within VPC boundaries and that are not directly reachable from internet. So this rules out PublicDnsNamespace support. Both PrivateDnsNamespace and HttpNamespace can be supported but given that most applications still use DNS resolution before making a connection to remote service (via Envoy), HttpNamespace cannot be readily used. In future, we plan on leveraging [Envoy's DNS filter](https://github.com/envoyproxy/envoy/issues/6748) to support both PrivateDnsNamespace and HttpNamespace seamlessly. For now it is required to create PrivateDnsNamespace to get both DNS resolution and App Mesh's EDS support. Note that both PrivateDnsNamespace and HttpNamespace services support custom attributes that can be used with DiscoverInstances API.
```

Comment:
Thanks for highlighting this. There is a better way to still use HttpNamespace but outside the scope of this work.